### ISSUE TITLE
Nothing happens with Razer Ornata Chroma

### ISSUE THREAD

**Issue by m1roh** [lah7/polychromatic#119]
The programm seems to work but I can try any option I want the keys still turned off... In the log file everything look like ok, I have really no idea…
Here an extract of razer.log : 

```
2017-08-29 17:26:11 | razer                          | INFO     | Initialising Daemon (v1.1.15). Pid: 3364
2017-08-29 17:26:11 | razer.screensaver              | INFO     | Initialising DBus Screensaver Monitor
2017-08-29 17:26:11 | razer                          | DEBUG    | Loaded device specification: RazerAbyssus ---------------------------- (1532:0042)
2017-08-29 17:26:11 | razer                          | DEBUG    | Loaded device specification: RazerAnansi ----------------------------- (1532:010F)
2017-08-29 17:26:11 | razer                          | DEBUG    | Loaded device specification: RazerBlackWidow2012 --------------------- (1532:010D)
2017-08-29 17:26:11 | razer                          | DEBUG    | Loaded device specification: RazerBlackWidow2013 --------------------- (1532:011A)
2017-08-29 17:26:11 | razer                          | DEBUG    | Loaded device specification: RazerBlackWidow2016 --------------------- (1532:0214)
2017-08-29 17:26:11 | razer                          | DEBUG    | Loaded device specification: RazerBlackWidowChroma ------------------- (1532:0203)
2017-08-29 17:26:11 | razer                          | DEBUG    | Loaded device specification: RazerBlackWidowChromaOverwatch ---------- (1532:0211)
2017-08-29 17:26:11 | razer                          | DEBUG    | Loaded device specification: RazerBlackWidowChromaTournamentEdition -- (1532:0209)
2017-08-29 17:26:11 | razer                          | DEBUG    | Loaded device specification: RazerBlackWidowChromaV2 ----------------- (1532:0221)
2017-08-29 17:26:11 | razer                          | DEBUG    | Loaded device specification: RazerBlackWidowClassic ------------------ (1532:011B)
2017-08-29 17:26:11 | razer                          | DEBUG    | Loaded device specification: RazerBlackWidowClassicAlternate --------- (1532:010E)
2017-08-29 17:26:11 | razer                          | DEBUG    | Loaded device specification: RazerBlackWidowXChroma ------------------ (1532:0216)
2017-08-29 17:26:11 | razer                          | DEBUG    | Loaded device specification: RazerBlackWidowXChromaTournamentEdition - (1532:021A)
2017-08-29 17:26:11 | razer                          | DEBUG    | Loaded device specification: RazerBlackWidowXUltimate ---------------- (1532:0217)
2017-08-29 17:26:11 | razer                          | DEBUG    | Loaded device specification: RazerBladeLate2016 ---------------------- (1532:0224)
2017-08-29 17:26:11 | razer                          | DEBUG    | Loaded device specification: RazerBladeProLate2016 ------------------- (1532:0210)
2017-08-29 17:26:11 | razer                          | DEBUG    | Loaded device specification: RazerBladeQHD --------------------------- (1532:020F)
2017-08-29 17:26:11 | razer                          | DEBUG    | Loaded device specification: RazerBladeStealth ----------------------- (1532:0205)
2017-08-29 17:26:11 | razer                          | DEBUG    | Loaded device specification: RazerBladeStealthLate2016 --------------- (1532:0220)
2017-08-29 17:26:11 | razer                          | DEBUG    | Loaded device specification: RazerChromaMugHolder -------------------- (1532:0F07)
2017-08-29 17:26:11 | razer                          | DEBUG    | Loaded device specification: RazerCore ------------------------------- (1532:0215)
2017-08-29 17:26:11 | razer                          | DEBUG    | Loaded device specification: RazerDeathStalkerChroma ----------------- (1532:0204)
2017-08-29 17:26:11 | razer                          | DEBUG    | Loaded device specification: RazerDeathStalkerExpert ----------------- (1532:0202)
2017-08-29 17:26:11 | razer                          | DEBUG    | Loaded device specification: RazerDeathadderChroma ------------------- (1532:0043)
2017-08-29 17:26:11 | razer                          | DEBUG    | Loaded device specification: RazerDeathadderElite -------------------- (1532:005C)
2017-08-29 17:26:11 | razer                          | DEBUG    | Loaded device specification: RazerDiamondbackChroma ------------------ (1532:004C)
2017-08-29 17:26:11 | razer                          | DEBUG    | Loaded device specification: RazerFireFly ---------------------------- (1532:0C00)
2017-08-29 17:26:11 | razer                          | DEBUG    | Loaded device specification: RazerImperator -------------------------- (1532:002F)
2017-08-29 17:26:11 | razer                          | DEBUG    | Loaded device specification: RazerKraken ----------------------------- (1532:0504)
2017-08-29 17:26:11 | razer                          | DEBUG    | Loaded device specification: RazerKrakenClassic ---------------------- (1532:0501)
2017-08-29 17:26:11 | razer                          | DEBUG    | Loaded device specification: RazerKrakenV2 --------------------------- (1532:0510)
2017-08-29 17:26:11 | razer                          | DEBUG    | Loaded device specification: RazerMamba2012Wired --------------------- (1532:0024)
2017-08-29 17:26:11 | razer                          | DEBUG    | Loaded device specification: RazerMamba2012Wireless ------------------ (1532:0025)
2017-08-29 17:26:11 | razer                          | DEBUG    | Loaded device specification: RazerMambaChromaTE ---------------------- (1532:0046)
2017-08-29 17:26:11 | razer                          | DEBUG    | Loaded device specification: RazerMambaChromaWired ------------------- (1532:0044)
2017-08-29 17:26:11 | razer                          | DEBUG    | Loaded device specification: RazerMambaChromaWireless ---------------- (1532:0045)
2017-08-29 17:26:11 | razer                          | DEBUG    | Loaded device specification: RazerNaga2014 --------------------------- (1532:0040)
2017-08-29 17:26:11 | razer                          | DEBUG    | Loaded device specification: RazerNagaHex ---------------------------- (1532:0041)
2017-08-29 17:26:11 | razer                          | DEBUG    | Loaded device specification: RazerNagaHexRed ------------------------- (1532:0036)
2017-08-29 17:26:11 | razer                          | DEBUG    | Loaded device specification: RazerNagaHexV2 -------------------------- (1532:0050)
2017-08-29 17:26:11 | razer                          | DEBUG    | Loaded device specification: RazerNostromo --------------------------- (1532:0111)
2017-08-29 17:26:11 | razer                          | DEBUG    | Loaded device specification: RazerOrbweaver -------------------------- (1532:0113)
2017-08-29 17:26:11 | razer                          | DEBUG    | Loaded device specification: RazerOrbweaverChroma -------------------- (1532:0207)
2017-08-29 17:26:11 | razer                          | DEBUG    | Loaded device specification: RazerOrnata ----------------------------- (1532:021F)
2017-08-29 17:26:11 | razer                          | DEBUG    | Loaded device specification: RazerOrnataChroma ----------------------- (1532:021E)
2017-08-29 17:26:11 | razer                          | DEBUG    | Loaded device specification: RazerOrochi2011 ------------------------- (1532:0013)
2017-08-29 17:26:11 | razer                          | DEBUG    | Loaded device specification: RazerOrochi2013 ------------------------- (1532:0039)
2017-08-29 17:26:11 | razer                          | DEBUG    | Loaded device specification: RazerOrochiWired ------------------------ (1532:0048)
2017-08-29 17:26:11 | razer                          | DEBUG    | Loaded device specification: RazerOuroboros -------------------------- (1532:0032)
2017-08-29 17:26:11 | razer                          | DEBUG    | Loaded device specification: RazerTaipan ----------------------------- (1532:0034)
2017-08-29 17:26:11 | razer                          | DEBUG    | Loaded device specification: RazerTartarus --------------------------- (1532:0201)
2017-08-29 17:26:11 | razer                          | DEBUG    | Loaded device specification: RazerTartarusChroma --------------------- (1532:0208)
2017-08-29 17:26:11 | razer                          | INFO     | Found device.0: 0003:1532:021E.0003
2017-08-29 17:26:11 | razer.device0                  | INFO     | Initialising device.0 RazerOrnataChroma
2017-08-29 17:26:11 | razer.device0                  | DEBUG    | Adding getSerial method to DBus
2017-08-29 17:26:11 | razer.device0                  | DEBUG    | Adding razer.device.misc.suspendDevice method to DBus
2017-08-29 17:26:11 | razer.device0                  | DEBUG    | Adding razer.device.misc.getDeviceMode method to DBus
2017-08-29 17:26:11 | razer.device0                  | DEBUG    | Adding razer.device.misc.getRazerUrls method to DBus
2017-08-29 17:26:11 | razer.device0                  | DEBUG    | Adding razer.device.misc.setDeviceMode method to DBus
2017-08-29 17:26:11 | razer.device0                  | DEBUG    | Adding razer.device.misc.resumeDevice method to DBus
2017-08-29 17:26:11 | razer.device0                  | DEBUG    | Adding razer.device.misc.getVidPid method to DBus
2017-08-29 17:26:11 | razer.device0                  | DEBUG    | Adding razer.device.misc.version method to DBus
2017-08-29 17:26:11 | razer.device0                  | DEBUG    | Adding razer.device.misc.hasDedicatedMacroKeys method to DBus
2017-08-29 17:26:11 | razer.device0                  | DEBUG    | Adding razer.device.misc.getFirmware method to DBus
2017-08-29 17:26:11 | razer.device0                  | DEBUG    | Adding razer.device.misc.getMatrixDimensions method to DBus
2017-08-29 17:26:11 | razer.device0                  | DEBUG    | Adding razer.device.misc.hasMatrix method to DBus
2017-08-29 17:26:11 | razer.device0                  | DEBUG    | Adding razer.device.misc.getDeviceName method to DBus
2017-08-29 17:26:11 | razer.device0                  | DEBUG    | Adding razer.device.misc.getDeviceType method to DBus
2017-08-29 17:26:11 | razer.device0                  | DEBUG    | Adding razer.device.lighting.brightness.getBrightness method to DBus
2017-08-29 17:26:11 | razer.device0                  | DEBUG    | Adding razer.device.lighting.brightness.setBrightness method to DBus
2017-08-29 17:26:11 | razer.device0                  | DEBUG    | Adding razer.device.lighting.chroma.setWave method to DBus
2017-08-29 17:26:11 | razer.device0                  | DEBUG    | Adding razer.device.lighting.chroma.setStatic method to DBus
2017-08-29 17:26:11 | razer.device0                  | DEBUG    | Adding razer.device.lighting.chroma.setSpectrum method to DBus
2017-08-29 17:26:11 | razer.device0                  | DEBUG    | Adding razer.device.lighting.chroma.setReactive method to DBus
2017-08-29 17:26:11 | razer.device0                  | DEBUG    | Adding razer.device.lighting.chroma.setNone method to DBus
2017-08-29 17:26:11 | razer.device0                  | DEBUG    | Adding razer.device.lighting.chroma.setBreathRandom method to DBus
2017-08-29 17:26:11 | razer.device0                  | DEBUG    | Adding razer.device.lighting.chroma.setBreathSingle method to DBus
2017-08-29 17:26:11 | razer.device0                  | DEBUG    | Adding razer.device.lighting.chroma.setBreathDual method to DBus
2017-08-29 17:26:11 | razer.device0                  | DEBUG    | Adding razer.device.lighting.chroma.setCustom method to DBus
2017-08-29 17:26:11 | razer.device0                  | DEBUG    | Adding razer.device.lighting.chroma.setKeyRow method to DBus
2017-08-29 17:26:11 | razer.device0                  | DEBUG    | Adding razer.device.led.gamemode.getGameMode method to DBus
2017-08-29 17:26:11 | razer.device0                  | DEBUG    | Adding razer.device.led.gamemode.setGameMode method to DBus
2017-08-29 17:26:11 | razer.device0                  | DEBUG    | Adding razer.device.led.macromode.getMacroMode method to DBus
2017-08-29 17:26:11 | razer.device0                  | DEBUG    | Adding razer.device.led.macromode.setMacroMode method to DBus
2017-08-29 17:26:11 | razer.device0                  | DEBUG    | Adding razer.device.led.macromode.getMacroEffect method to DBus
2017-08-29 17:26:11 | razer.device0                  | DEBUG    | Adding razer.device.led.macromode.setMacroEffect method to DBus
2017-08-29 17:26:11 | razer.device0                  | DEBUG    | Adding razer.device.macro.getMacros method to DBus
2017-08-29 17:26:11 | razer.device0                  | DEBUG    | Adding razer.device.macro.deleteMacro method to DBus
2017-08-29 17:26:11 | razer.device0                  | DEBUG    | Adding razer.device.macro.addMacro method to DBus
2017-08-29 17:26:11 | razer.device0                  | DEBUG    | Adding razer.device.lighting.chroma.setStarlightRandom method to DBus
2017-08-29 17:26:11 | razer.device0                  | DEBUG    | Adding razer.device.lighting.chroma.setStarlightSingle method to DBus
2017-08-29 17:26:11 | razer.device0                  | DEBUG    | Adding razer.device.lighting.chroma.setStarlightDual method to DBus
2017-08-29 17:26:11 | razer.device0                  | DEBUG    | Adding razer.device.lighting.custom.setRipple method to DBus
2017-08-29 17:26:11 | razer.device0                  | DEBUG    | Adding razer.device.lighting.custom.setRippleRandomColour method to DBus
2017-08-29 17:26:11 | razer.device0.keymanager       | DEBUG    | Starting KeyWatcher
2017-08-29 17:26:11 | razer.device0                  | INFO     | Putting device into driver mode. Daemon will handle special functionality
2017-08-29 17:26:11 | razer                          | INFO     | Adding razer.devices.getDevices method to DBus
2017-08-29 17:26:11 | razer                          | INFO     | Adding razer.daemon.supportedDevices method to DBus
2017-08-29 17:26:11 | razer                          | INFO     | Adding razer.devices.enableTurnOffOnScreensaver method to DBus
2017-08-29 17:26:11 | razer                          | INFO     | Adding razer.devices.syncEffects method to DBus
2017-08-29 17:26:11 | razer                          | INFO     | Adding razer.devices.syncEffects method to DBus
2017-08-29 17:26:11 | razer                          | INFO     | Adding razer.devices.syncEffects method to DBus
2017-08-29 17:26:11 | razer                          | INFO     | Adding razer.daemon.version method to DBus
2017-08-29 17:26:11 | razer                          | INFO     | Adding razer.daemon.stop method to DBus
2017-08-29 17:26:11 | razer                          | INFO     | Serving DBus
2017-08-29 17:26:13 | razer                          | DEBUG    | DBus called get_serial_list
2017-08-29 17:26:13 | razer.device0                  | DEBUG    | DBus call get_device_type
2017-08-29 17:26:13 | razer.device0                  | DEBUG    | DBus call get_device_name
2017-08-29 17:26:13 | razer.device0                  | DEBUG    | DBus call get_device_type
2017-08-29 17:26:13 | razer.device0                  | DEBUG    | DBus call get_firmware
2017-08-29 17:26:13 | razer.device0                  | DEBUG    | DBus call version
2017-08-29 17:26:13 | razer.device0                  | DEBUG    | DBus call has_matrix
2017-08-29 17:26:13 | razer.device0                  | DEBUG    | DBus call has_matrix
2017-08-29 17:26:14 | razer.device0                  | DEBUG    | DBus call get_brightness
2017-08-29 17:26:14 | razer.device0                  | DEBUG    | DBus call get_brightness
2017-08-29 17:27:00 | razer.device0                  | DEBUG    | DBus call set_brightness
2017-08-29 17:27:00 | razer.device0                  | DEBUG    | Sending observer message: ('effect', RazerOrnataChroma:SI1704F12003442, 'setBrightness', 255)
2017-08-29 17:27:00 | razer.device0                  | DEBUG    | DBus call get_brightness
2017-08-29 17:27:00 | razer.device0                  | DEBUG    | DBus call get_brightness
2017-08-29 17:27:06 | razer.device0                  | DEBUG    | DBus call set_wave_effect
2017-08-29 17:27:06 | razer.device0                  | DEBUG    | Sending observer message: ('effect', RazerOrnataChroma:SI1704F12003442, 'setWave', dbus.Int32(1))
2017-08-29 17:27:06 | razer.device0                  | DEBUG    | DBus call get_brightness
2017-08-29 17:27:06 | razer.device0                  | DEBUG    | DBus call get_brightness
2017-08-29 17:27:08 | razer.device0                  | DEBUG    | DBus call set_wave_effect
2017-08-29 17:27:08 | razer.device0                  | DEBUG    | Sending observer message: ('effect', RazerOrnataChroma:SI1704F12003442, 'setWave', dbus.Int32(2))
2017-08-29 17:27:08 | razer.device0                  | DEBUG    | DBus call get_brightness
2017-08-29 17:27:09 | razer.device0                  | DEBUG    | DBus call set_wave_effect
2017-08-29 17:27:09 | razer.device0                  | DEBUG    | Sending observer message: ('effect', RazerOrnataChroma:SI1704F12003442, 'setWave', dbus.Int32(1))
2017-08-29 17:27:09 | razer.device0                  | DEBUG    | DBus call get_brightness
2017-08-29 17:27:09 | razer.device0                  | DEBUG    | DBus call get_brightness
2017-08-29 17:30:19 | razer                          | DEBUG    | DBus called get_serial_list
2017-08-29 17:30:19 | razer.device0                  | DEBUG    | DBus call get_device_type
2017-08-29 17:30:19 | razer.device0                  | DEBUG    | DBus call get_device_name
2017-08-29 17:30:19 | razer.device0                  | DEBUG    | DBus call get_device_type
2017-08-29 17:30:19 | razer.device0                  | DEBUG    | DBus call get_firmware
2017-08-29 17:30:19 | razer.device0                  | DEBUG    | DBus call version
2017-08-29 17:30:19 | razer.device0                  | DEBUG    | DBus call has_matrix
2017-08-29 17:30:19 | razer.device0                  | DEBUG    | DBus call has_matrix
2017-08-29 17:30:20 | razer.device0                  | DEBUG    | DBus call get_brightness
2017-08-29 17:30:20 | razer.device0                  | DEBUG    | DBus call get_brightness
```

Is anyone have the same issue?

**Comment by lah7** [comment]
So, am I correct in thinking that when you try to change effects or brightness, nothing happens at all?

**Comment by m1roh** [comment]
Yes exactly, the only thing that works is enabling the game mode, so the keyboard is detected...

**Comment by lah7** [comment]
I'm guessing it's very likely to be daemon-level, where the daemon "thinks" it's working, but the driver is sending the wrong commands to the keyboard (possibly different model, different firmware, etc), so nothing happens.

It's out of Polychromatic's hands, as all it does is tell the daemon to do things which it is being told is done successfully.

Pinging @terrycain (driver/daemon guy) who might know what's up with this device.

----------
In the meantime, what is the PID of this device?

    lsusb | grep 1532

**Comment by m1roh** [comment]
I finally solved the problem. In fact the last time I turned off my computer before deleting Windows, I was logged on my razer account with my "non-lighting profile". So it's look like something, somewhere, remembered that. So I just pluggged my keyboard on a windows computer, installed the official driver, logged on my razer synapse account and move on my "lighting profile", then I reppluged this on my ubuntu laptop and... magic happens ! It works !!!!! Thank you for your answers...
